auto_nag/scripts/duplicate_copy_metadata.py

```diff
@@ -55,35 +56,49 @@ def get_bugs(self, date="today", bug_ids=[], chunk_size=None):
                 if not dup_bug:
                     continue
 
+                # Performance Impact: copy the assessment result from duplicates
+                if bug.get("cf_performance_impact") == "---" and dup_bug[
+                    "cf_performance_impact"
+                ] not in ("---", "?"):
+                    field_label = "Performance Impact"
+                    if field_label not in copied_fields:
+                        copied_fields[field_label] = {
+                            "from": [dup_bug["id"]],
+                            "value": dup_bug["cf_performance_impact"],
+                        }
+                    else:
+                        copied_fields[field_label]["from"].append(dup_bug["id"])
+
                 # Keywords: copy the `access` keyword from duplicates
                 if "access" not in bug["keywords"] and "access" in dup_bug["keywords"]:
-                    if "keywords" not in copied_fields:
-                        copied_fields["keywords"] = {
+                    field_label = "Keywords"
```

Comment:
Nit: I'd keep it as before, it's ok to repeat "keywords" given that it is kind of a field name of an object.

If you prefer to keep it like this, the "K" should be "k".

Reply:
Forget the second part, you're not accessing `bug` with `field-label` :)

Reply:
> Nit: I'd keep it as before, it's ok to repeat "keywords" given that it is kind of a field name of an object.

I agree with you. I like it as before. I did the refactoring in 9cddf31271a59cbf6f3462f389041dbe1ff654db to make it easy to show labels for complix field names. For example, the field label for `cf_performance_impact` is `Performance Impact`.



> If you prefer to keep it like this, the "K" should be "k".

It is "k" for the field name and "K" for the field label. The only difference between the field name and the field label in this case is the capitalization, for other fields, it could be more than that (e.g., `cf_performance_impact` -->  `Performance Impact`).

> Forget the second part, you're not accessing `bug` with `field-label` :)


The field label is what we show in the comment/email and Bugzilla shows in the UI, where the field name is what we use to access the field from the `bug` object.


If you find this confusing I could use a different approach to do the transformation between the field name and field label (e.g., use a map) or make it more explicit by adding a new variable called `field_name`. 

@marco-c WDYT?

Reply:
Since that was not super obvious and required much explanation, I reverted the refactoring in 06e9771ec05479521c4d72c32985d20e125ec3ab and replaced it with maps in 6866213ac91bf5a2368bc18c546fee0b2a7d5cc6.